source/getting_started.rst

```diff
@@ -1,5 +1,9 @@
 .. _getting_started:
 
+.. Note:: 
+	Adam:
+	see Index notes on expanding on what WEC-Sim is and can do. Merge with index into an Introduction section?
+
```

Comment:
@H0R5E had a similar comment. I recommend reviewing https://github.com/WEC-Sim/WEC-Sim/pull/369 for similarity and coordinating with @H0R5E on how to improve/merge those sections.

Reply:
@kmruehl I am going to remove this note and create a distinct project for it as it would be substantial, and too much with this current PR.